connector/mqtt/locust/README.md

```diff
@@ -275,76 +181,57 @@ DOJOT_MQTT_TIMEOUT      | MQTT broker timeout        | 60            | integer
-DOJOT_URL             | dojot instance address               | http://127.0.0.1:8000 | hostname/IP        |
-DOJOT_USER            | dojot user                           | admin                 | usernames          |
-DOJOT_API_RETRIES     | number of retries for API calls      | 3                     | integer            |
-DOJOT_API_RETRY_TIME  | time to wait between retries         | 5000                  | float time in ms   |
+Key                     | Purpose                              | Default Value         | Valid Values       |
+----------------------- | ------------------------------------ | --------------------- | ------------------ |
+DOJOT_API_RETRIES       | number of retries for API calls      | 3                     | integer            |
+DOJOT_API_RETRY_TIME    | time to wait between retries         | 5000                  | float time in ms   |
+DOJOT_DEVICES_PAGE_SIZE | /device endpoint page size           | 20                    | positive integers  |
+DOJOT_ENV               | use a dojot instance                 | n                     | y, n               |
+DOJOT_GATEWAY_TIMEOUT   | dojot auth API timeout               | 180                   | integer            |
+DOJOT_PASSWD            | dojot user's password                | admin                 | passwords          |
+DOJOT_URL               | dojot instance address               | http://127.0.0.1:8000 | hostname/IP        |
+DOJOT_USER              | dojot user                           | admin                 | usernames          |
 
 ## **Operating System**
 
 While small tests can be run without problems, bigger ones create some obstacles.
 To create a lot of clients in only one machine, the default number of ports in the
-OS will not accomodate the required number of connections. To increase it, run:
+OS will not accommodate the required number of connections. To increase it, run:
 
 ```shell
 sudo sysctl -w net/ipv4/ip_local_port_range="1024 65535"
 ```
 
 # Development
 
+You can use the Docker Compose provided with the `generate_certs` script to develop. In case you
+missed it, you can see it [here](#generate-certificates).
+
 ## Lint
 
-There is a lint in the repository that you should follow. To check the linting state in it, run:
+There is a lint in the repository that you should follow. To check the linting state in it:
 ```shell
 run_lint
 ```
 
-The command `run_lint` is an alias of `pylint src --rcfile=.pylintrc tests --rcfile=.pylintrc`.
+If you are not running inside the generate_certs Docker Compose, run:
+```shell
+pylint src --rcfile=.pylintrc tests --rcfile=.pylintrc
+```
 
 ## Unit Tests
 
-To run the tests, you can use the Docker Compose file provided for use with the `generate_certs` script. In case you missed it, you can see it [here](#generate-certificates).
-With the dependencies properly installed, you can get the complete tests' report by running:
+You can get the complete tests' report by running:
```

Comment:
> ou can get the complete **tests'** report by running:

Is this apostrophe intended to truncate the word `test's` or does it have another purpose?

Reply:
It's the same as `test's`, but for the plural you don't add the `s` at the end. But it is kinda strange here, I will remove it.